index.html

```diff
@@ -142,30 +152,31 @@ <h1>Tonic</h1>
             </div>
 
         </div>
-        <div class="card-work">
-            <img class="work-presentation" src="css/img/Snapshoot4.png" >
+        <div class="card-work-par">
+            <img class="work-presentation-desktop" src="css/img/Portoflio Card/desktop4.png">
+            <img class="work-presentation-mobile" src="css/img/Snapshoot4.png" >
             <div class="block">
                 <div class="first-text-section">
-                    <h1>Multi-Post <br>
-                        Stories
+                    <h1>
+                        Uber Navigation
                     </h1>
                     <div class="frame">
                         <ul>
-                            <li class="frame1">CANOPPY</li>
+                            <li class="frame1">Uber</li>
                             <li><img class="dot" img src="css/img/Counter.png"></li>
-                            <li class="frame2">Back End Dev</li>
+                            <li class="frame2">Lead Developer</li>
                             <li><img class="dot" img src="css/img/Counter.png"></li>
-                            <li class="frame3">2015</li>      
+                            <li class="frame3">2018</li>      
                         </ul>
                     </div>
                 </div>
                 <p class="primary-text"> 
-                    A daily selection of privately personalized reads;
-                    no accounts or sign-ups required.
+                    A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.

```

Comment:
- [ ] Great job so far. Let's make some final touches. Please, kindly add the top right radius to the about section, so it can align well with the original design. this will also help you learn more.
yours
![Screenshot (1143)_LI](https://github.com/BiancaMe/Portfolio-Mobile/assets/65298048/46b54fcc-8835-43bb-92d6-c3361e0a5b29)
original design
![Screenshot (1144)_LI](https://github.com/BiancaMe/Portfolio-Mobile/assets/65298048/038d2b46-c09d-4f47-871a-37d2c928d58f)